client/src/components/Card.js

```diff
@@ -108,33 +109,24 @@ const StockInputContainer = styled.div`
 export default function Card({ title, price, symbol, stock, id }) {
   const [stockValue, setStockValue] = useState(stock);
   const [, , updateUserCardData] = useContext(CoinContext);
```

Comment:
Why are there two commas?

Reply:
This is because I get multiple items from that context. And if I want only the third item, like in this case, I need to skip the first two items.

Reply:
Ah, learned something again! :)